src/App.js

```diff
@@ -7,8 +7,14 @@ class App extends Component {
     return (
       <div className="App">
         <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
+          <img src={logo} className="App-logo" alt="logo" onClick={()=>this.props.history.push("")}/>
           <h1 className="App-title">Crick live</h1>
+          <button type="button" className="btn-login button" name="login" onClick={()=>this.props.history.push('/login')}>
```

Comment:
Instead of calling this.props.history.push directly, write a high level function and pass the param '/login'. Always try to write small small reusable functions which will make the code more readable and simple.